projects/boringssl/Dockerfile

```diff
@@ -15,7 +15,10 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN apt-get update && apt-get install -y cmake ninja-build golang
-
+RUN apt-get update && apt-get install -y cmake ninja-build golang curl wget \
+    binutils cmake ninja-build liblzma-dev libz-dev pkg-config autoconf libtool
 RUN git clone --depth 1 https://boringssl.googlesource.com/boringssl
-COPY build.sh $SRC/
+RUN git clone --depth 1 https://github.com/dannyallover/fuzzing
```

Comment:
I think we'll get your change in the google/fuzzing repo landed sooner that this OSS-Fuzz PR, so we should replace this URL later but before merging this

Reply:
please also add a comment before this line like "# Use ASN.1 proto from the google/fuzzing repo.`

Reply:
@dannyallover Have you had a chance to address this yet? :)